test-packages/test-services-odata-v2/test-service/TestEntityLvl2SingleLink.ts

```diff
@@ -9,6 +9,7 @@ import {
   DeSerializers,
   DeserializedType
 } from '@sap-cloud-sdk/odata-v2';
+import { camelCase } from '@sap-cloud-sdk/util';
```

Comment:
Yes, this is not on purpose. The change is in `packages/generator/src/entity/file.ts`, but I think after our discussions on how multi-key entities should be handled this will go anyway from the generated code.